modules/storage/storage-account/version.json

```diff
@@ -1,6 +1,6 @@
 {
   "$schema": "https://aka.ms/bicep-registry-module-version-file-schema#",
-  "version": "2.0",
+  "version": "3.0",
```

Comment:
would like to understand what are the particular breaking changes that we are making. In the PR description, could you include something like an "upgrade guide" to explain to me what changes I would need to make to go from 2->3? 

(i dont normally ask for this, but the storage module is the one I use the most in the most places and I can't clearly tell from just the PR)

Reply:
There are a lot of variables added, updated/renamed, and removed. It should be a major/breaking change.

Reply:
I will create a `upgrade guide` soon.

Reply:
@thllxb , haven't seen upgrade directions yet? Particularly what parameters do you want to remove. That's really all im concerned about.

Reply:
@dciborow Please see the updated PR description. It now contains the `Breaking Changes` section where all breaking changes are listed.